react/components/UserAddress/ChangeAddressIcon.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const ChangeAddressIcon = () => {
+  return (
+    <svg width="21" height="27" viewBox="0 0 21 27" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M19.9655 10.4828C19.9655 16.3448 10.4828 26 10.4828 26C10.4828 26 1 16.431 1 10.4828C1 4.44828 5.91379 1 10.4828 1C15.0517 1 19.9655 4.53448 19.9655 10.4828Z" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
shouldn't `stroke-width`, `stroke-miterlimit`, `stroke-linecap` and `stroke-linejoin` be capitalized like `strokeWidth`?

Reply:
It works like this, so 🤷‍♂️

Reply:
I think react warns if the props are like that, but I guess it's fine